scripts/lib/pb_core_ingester.rb

```diff
@@ -81,14 +83,17 @@ def ingest_xml_no_commit(xml)
       raise ValidationError.new(e)
     end
 
-    begin
-      @solr.add(pbcore.to_solr)
-    rescue => e
-      raise SolrError.new(e)
+    if @re.match(xml)
```

Comment:
Actually, looking at it again, it could be a bit faster if we didn't bother to xml parse the ones that don't match. I could do that if you want...